pkg/volume/util/device_util.go

```diff
@@ -0,0 +1,34 @@
+/*
+Copyright 2015 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package util
+
+//DeviceUtil is a util for common
+//it also backports certain operations from golang 1.5
+type DeviceUtil interface {
+	FindMultipathDeviceForDevice(disk string) string
+	FindDevicesForMultipathDevice(disk string) []string
```

Comment:
It was needed before some changes landed to iscsi.
Shall I leave it in as a utility function or remove?